src/Polly.Core/Polly.Core.csproj

```diff
@@ -11,12 +11,8 @@
     <LegacySupport>true</LegacySupport>
     <InjectSharedSources>true</InjectSharedSources>
   </PropertyGroup>
-  <PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">
```

Comment:
So no AOT on .NET 6

Reply:
It just isn't being marked as it is explicitly, but in the #2003 PR with the .NET 9 SDK if you enable it it errors out saying it's not supported.

Either I can turn it back on, but we'll lose it again in November, or we turn it off now.

Reply:
Ahh ok then :)